src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -827,4 +827,10 @@ Do you want to proceed?</value>
   <data name="Change_configuration" xml:space="preserve">
     <value>Change configuration</value>
   </data>
+  <data name="Filter" xml:space="preserve">
+    <value>Filter</value>
+  </data>
+  <data name="Warning" xml:space="preserve">
+    <value>Warning</value>
```

Comment:
curious what these two localized strings are for. it's not immediately obvious from their names/values.

Reply:
Note: because of htat, we likely need to have a note provided to the loc people so they have context as well.

Reply:
These are automation names for image elements. The editor said each team should maintain own names even standard independently.

---
In reply to: [222071213](https://github.com/dotnet/roslyn/pull/29016#discussion_r222071213) [](ancestors = 222071213)

Reply:
I hope that extending this to *__image_element_automation_name is enough for the context. Please let me know if we should provide a better one.

---
In reply to: [223927682](https://github.com/dotnet/roslyn/pull/29016#discussion_r223927682) [](ancestors = 223927682)

Reply:
@ivanbasov There's a "note" or "comment" syntax (I forget what, exactly) and that can be passed off. Small update there?